File: pyramid.py
x = int(input("Enter x: "))

odd_numbers = []
number = 1

while len(odd_numbers) < x:
    if number % 2 !=0 :
        odd_numbers.append(number)
    number+=1


for i in range(x):
    print(" ".join(map(str, odd_numbers[:i + 1])))

File: pyramid.js
const x = parseInt(prompt("Enter x: "),10)

let oddNumbers = [];

let number = 1;

while (oddNumbers.length < x){
    if (number % 2 !== 0){
        oddNumbers.push(number);
    }
    number++;
}

for (let i = 0; i< x; i++){
    let rows= oddNumbers.slice(0,i+1).map(String)
    console.log(rows.join(" "));
}